src/apps/loyalty/server/index.tsx

```diff
@@ -69,6 +70,18 @@ app.get("/inquiries", (req, res) => {
     return res.redirect(req.baseUrl + "/login")
   }
 
+  const headers = { "X-Access-Token": req.user.get("accessToken") }
```

Comment:
@l2succes check out the commit I just pushed, we don't really need that model after all.